sideboard/tests/test_profiler.py

```diff
@@ -0,0 +1,16 @@
+from sideboard.lib import cleanup_profiler, profile
+from sideboard.config import config
+
+
+def some_function():
+    pass
+
+
+def test_profile_is_noop(monkeypatch):
```

Comment:
The only thing I am testing is that the `@profile` decorator is a no-op when profiling is disabled. This is the only code that will be running in production. Everything else in the profiler module is per se testing code.